Gruntfile.js

```diff
@@ -20,30 +20,12 @@ module.exports = function(grunt) {
       }
     },
     concat: {
-      lawnchair: {
```

Comment:
This is no longer required thanks to browserify.

Reply:
Yeah. This is just leftover from js-sdk. Sadly we need to still bundle lawnchair as source code weren't released for the last couple years and we made some fixes directly upstream to maintain it :)